src/Speck.js

```diff
@@ -134,13 +134,32 @@ class Speck {
     return JSON.parse(JSON.stringify(rawData));
   }
 
+  toJSONContext(context) {
```

Comment:
I firstly named it `toJSONByContext` (as you can see in one of my commits), but it felt weird, since it is `validateContext`, not `validateByContext`, so I refactored... but it's easily changeable 😁